bodhi/tests/server/services/test_releases.py

```diff
@@ -291,7 +294,8 @@ def test_query_releases_html_two_releases_same_state(self):
             "pending_signing_tag": "f42-updates-testing-signing",
             "pending_testing_tag": "f42-updates-testing-pending",
             "override_tag": "f42-override",
-            "csrf_token": self.get_csrf_token()}
+            "csrf_token": self.get_csrf_token(),
+            "mail_template": "fedora_errata_template"}
```

Comment:
We need a test that shows that mail template can be changed via the API and that it alters the db.